lib/helpers/error-redirect.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
`express-stormpath` doesn't have any tests for the `jsonError` function which is what the `error-redirect` is copied from. There's really nothing to test -- it just redirects.